### PR TITLE
Dev Portal Helium Flows

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
This PR represents an update of the Developer Portal documentation. 

[JIRA Ticket](https://konghq.atlassian.net/browse/DOCU-2246?jql=project%20%3D%20DOCU%20AND%20status%20%3D%20Backlog%20AND%20fixVersion%20%3D%20Helium)

# Preview 
https://deploy-preview-3897--kongdocs.netlify.app/
# Questions

- Does the troubleshooting section in the current state need to exist? 

## todo: 
- [x] Update Instructions
- [x] Restructure and condense docs
- [ ] Review Docs

### CODE REVIEW

File: app/konnect/dev-portal/publish.md [hunk 1]
```diff
@@ -3,34 +3,63 @@ title: Publish a Service to the Dev Portal
...
+The Developer Portal is accessible by a default URL. For instructions on customizing the URL of your Developer Portal, please see our [Customization Reference](/konnect/dev-portal/customization/custom/). 
+
+## Enable or disable public access for a Dev Portal {#access}
+
+<<<<<<< HEAD
+1. From the left navigation menu, open the **ServiceHub** page and select a Service.
+=======
+1. In {{site.konnect_short_name}}, open {% konnect_icon dev-portal %}
+**Dev Portal** from the left side menu, then click **Settings**.
+>>>>>>> 7234f32b1e (Add troubleshooting section)
```
Comment (lena-larionova) [inline app/konnect/dev-portal/publish.md:57]: Looks like a missed conflict.
```suggestion
1. In {{site.konnect_short_name}}, open {% konnect_icon dev-portal %}
**Dev Portal** from the left side menu, then click **Settings**.
```

File: app/konnect/dev-portal/applications/manage-app-connections.md [hunk 2]
```diff
@@ -91,16 +89,18 @@ that was revoked for a Service can be approved again any time at your discretion
 
 3. In the row for Service whose connection status you want to change, click the
 icon and choose **Approve** from the context menu.
+=======
+>>>>>>> 0b07779723 (Dev Portal)
```
Comment (lena-larionova) [inline app/konnect/dev-portal/applications/manage-app-connections.md:93]:
```suggestion
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 3]
```diff
@@ -50,66 +37,48 @@ permissions.
...
 
-   3. (Required) Enter the claim in the **Consumer claims** field.
+## Enable App Registration with OpenID Connect {#oidc-flow}
 
-   4. (Required) Enter one or more auth methods in the **Auth method** field.
+<<<<<<< HEAD
+1. From the {{site.konnect_short_name}} menu, click **ServiceHub** and select a
+=======
+1. From the {{site.konnect_short_name}} menu, click {% konnect_icon servicehub %} **ServiceHub** and select a
+>>>>>>> 7234f32b1e (Add troubleshooting section)
```
Comment (lena-larionova) [inline app/konnect/dev-portal/applications/enable-app-reg.md:70]:
```suggestion
1. From the {{site.konnect_short_name}} menu, click {% konnect_icon servicehub %} **ServiceHub** and select a
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 4]
```diff
@@ -50,66 +37,48 @@ permissions.
...
-1. From the {{site.konnect_short_name}} menu, click {% konnect_icon servicehub %}
-**ServiceHub** and select a Service.
+## Enable App Registration with Key Authentication {#konnect-key-auth-flow}
 
-1. From the **Actions** dropdown menu, select **Enable app registration**.
+<<<<<<< HEAD
+1. From the {{site.konnect_short_name}} menu, click **ServiceHub** and select a
+=======
+1. From the {{site.konnect_short_name}} menu, click {% konnect_icon servicehub %}**ServiceHub** and select a
+>>>>>>> 7234f32b1e (Add troubleshooting section)
```
Comment (lena-larionova) [inline app/konnect/dev-portal/applications/enable-app-reg.md:46]:
```suggestion
1. From the {{site.konnect_short_name}} menu, click {% konnect_icon servicehub %}**ServiceHub** and select a
```

**Comment by Guaris** [comment]
Closed in favor of #3918

**Pull request closed without merging**